evaluator/src/test/kotlin/PackageRuleTest.kt

```diff
@@ -0,0 +1,127 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.evaluator
+
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.WordSpec
+
+class PackageRuleTest : WordSpec() {
+    private val ruleSet = RuleSet(ortResult)
+
+    init {
+        "hasLicense()" should {
+            "return true if the package has concluded licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithOnlyConcludedLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return true if the package has declared licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithOnlyDeclaredLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return true if the package has detected licenses" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, licenseFindings)
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package has no license" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.hasLicense()
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isExcluded()" should {
+            "return true if the package is excluded" {
+                val rule = PackageRule(ruleSet, "test", packageExcluded, emptyList())
+                val matcher = rule.isExcluded()
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package is not excluded" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isExcluded()
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isFromOrg()" should {
+            "return true if the package is from org" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isFromOrg("here")
+
+                matcher.matches() shouldBe true
+            }
+
+            "return false if the package is not from org" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
+                val matcher = rule.isFromOrg("unknown")
+
+                matcher.matches() shouldBe false
+            }
+        }
+
+        "isType()" should {
+            "return true if the package has the provided type" {
+                val rule = PackageRule(ruleSet, "test", packageWithoutLicense, emptyList())
```

Comment:
Can we make it obvious that the package matched against actually is a Maven package?